Assets/Scripts/UI/PickupPup/DogBrowser/DogBrowser.cs

```diff
@@ -81,6 +81,7 @@ public void Open(int pageIndex = NONE_VALUE)
 
 	public void Close()
```

Comment:
I think I wrote this, but on review, I don't see why this isn't just an override of the `Hide` method. Seems cleaner